### PR TITLE
valdas page added

### PR DESCRIPTION
**Pull request by MrStonkus**

### CODE REVIEW

File: valdas.html [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Valdas Stonkus</title>
```
Comment (kstulgys) [inline valdas.html:8]: Well done that you changed this

File: valdas.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+</head>
+
+<body>
+  <h2>Hello world, this is Valdas. I am beginner developer, do not push me a lot ;)</h2>
+
```
Comment (kstulgys) [inline valdas.html:13]: Please remove this line

**Review by kstulgys — CHANGES_REQUESTED** [review]

**Review by seinauskaite — APPROVED** [review]
💯

**Review by SwallowEve — APPROVED** [review]
Nice work

File: valdas.html [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Valdas Stonkus</title>
+</head>
+
+<body>
+  <div>
+    <h2 style="text-align: center">Hello world, here is Valdas. I am a beginner developer, don't press me too much;)
+    </h2>
+    <div style="text-align: center">
+      <img src="./src/valdas_stonkus_171x171px.jpg" alt="Valdas Stonkus' picture">
```
Comment (eciras) [inline valdas.html:16]: Link doesn't work.

**Review by eciras — APPROVED** [review]

**Review by MarcinkEve — APPROVED** [review]

**Review by MartynasKupsys — APPROVED** [review]
Good work!! :)

File: valdas.html (hunk 3 above)
Comment (MrStonkus) [reply valdas.html:16]: > 

What link? There is no links :)

**Review by seinauskaite — APPROVED** [review]
good

File: valdas.html (hunk 2 above)
Comment (MrStonkus) [reply valdas.html]: Changes made, review please.

**Review by kstulgys — APPROVED** [review]

**Pull request merged**